dictionary/templates/dictionary/search.html

```diff
@@ -6,14 +6,16 @@
 
 {% block body %}
 <div class="container">
-    <form method="POST" action="" >
+    <form method="GET" action="" >
         {% csrf_token %}
         {% bootstrap_form form %}
         <button type="submit" class="btn btn-primary">Search</button>
     </form>
+</div>
 
     <hr />
-    {% if results%}
+<div class="container">
```

Comment:
Thanks.  I need to read the bootstrap docs a bit.  I'll fix this on the next go around.